src/API/AccessionTable.js

```diff
@@ -0,0 +1,60 @@
+import { useQueryClient } from 'react-query';
+
+import { API_PATH } from '../const';
```

Comment:
Could you probably start using `useNamespace` in place of stuff like this [now that it's implemented](https://github.com/folio-org/stripes-core/pull/1061)? Maybe even stuff the call to it inside `useOkapiQuery` so you never have to write it again? 🤔

Reply:
Yeah, I thought of it. But it returns something like `@folio/remote-storage`, and I need `API_PATH` to be just `remote-storage`.
So I probably need to `.split('/')` or something...
So finally I decided not to bother with it for now.

`useOkapiQuery` is used not only for module-level API requests (e.g. here I use global `/locations`), so hard-wiring it there would probably be a problem